packages/calypso-e2e/src/lib/pages/themes-detail-page.ts

```diff
@@ -23,4 +33,40 @@ export class ThemesDetailPage extends BaseContainer {
 		await this.page.click( selectors.demoPane );
 		await PreviewComponent.Expect( this.page );
 	}
+
+	/**
+	 * Activates the theme.
+	 *
+	 * If the optional parameter `keepModal` is set to true, the Thanks message modal will not
+	 * be dismissed.
+	 *
+	 * @param {boolean} [keepModal] Optional parameter that if set to true will keep the thanks modal open. Defaults to false.
+	 * @returns {Promise<void>} No return value.
+	 */
+	async activate( { keepModal = false }: { keepModal?: boolean } = {} ): Promise< void > {
+		await this.page.click( selectors.activateDesignButton );
+		await this.page.waitForSelector( selectors.activateModal );
+		await this.page.click( selectors.activateModalButton );
+		await this.page.waitForSelector( selectors.thanksMessage );
+		if ( ! keepModal ) {
+			await this.page.keyboard.press( 'Escape' );
```

Comment:
I chose to use the ESCAPE key to dismiss the modal instead of alternative (eg. clicking on the overlay outside of the modal). 

Hard to say what users do more often, but I suspect this isn't too important in the grand scheme of things.

Reply:
I like that! It seems a cleaner, more stable option than trying to find the right spot in the overlay to click for sure.